iavl_orphaning_tree.go

```diff
@@ -75,9 +79,7 @@ func (tree *orphaningTree) SaveVersion(version uint64) {
 	tree.ndb.SaveBranch(tree.root, func(node *IAVLNode) {
 		// The node version is set here since it isn't known until we save.
 		node.version = version
-		node.hashWithCount()
-
-		tree.Unorphan(node.hash)
+		tree.Unorphan(node._hash())
```

Comment:
okay, so _hash() is still called twice.
but it is not recursive and it caches the results, so that is fine with me

Reply:
Cool.